### DISCUSSION TITLE
SNES: ERROR Rom header corrupt or missing

### DISCUSSION THREAD

**Discussion by rrig** [sanni/cartreader#139] [Need help]
When I select cart type I get an error message, press button for manual config.
Press for manual config get "Name: unknown, Type: LoROM 15.
press button, Choose Mapping, I choose LoRom 256K
Next screen- Read Rom, Process starts Saving LoRom.
Checksum error D2D0.
The Game cart is Mega Man X and works in the SNES console.
I've included a pic of rom info from emulator
Thanks for any help you can provide.

![2_Error rom header corrupt or missing](https://user-images.githubusercontent.com/17235119/116567900-0d079300-a8d6-11eb-9b3a-2c18c9290f04.jpg)
![3_Name unknown lorom](https://user-images.githubusercontent.com/17235119/116569444-645a3300-a8d7-11eb-8fdc-fcc8f2ac1fe1.jpg)
![Rom info on emulator](https://user-images.githubusercontent.com/17235119/116569034-04638c80-a8d7-11eb-9ebd-3333d40df850.PNG)

**Comment by sanni** [comment]
Either the cart is dirty, use a q-tip with 99% IPA and wipe the black oxidation layer off the gold fingers, or there is a short or cold solder joint somewhere on the Cart Reader PCB.

**Reply by rrig** [reply]
Thanks for the quick reply.
